src/components/app/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import AllUsers from '../../containers/AllUsers';
+import User from '../../containers/User';
+import Post from '../../containers/Post';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Home from '../../containers/LongestPosts';
+import AllPosts from '../../containers/AllPosts';
+
+export default function App() {
+  return (
+    <>
```

Comment:
```suggestion
    <Router>
      <>
        <Header/>
        <Switch>
          <Route exact path="/" component={Home} />
          <Route exact path="/users/:id" component={User} />
          <Route exact path="/users" component={AllUsers} />
          <Route exact path="/posts" component={AllPosts} />
          <Route exact path="/posts/:id" component={Post} />
        </Switch>
      </>
    </Router>
```
Maybe try this configuration so that you have access to the nav bar across pages.